webservice/src/main/java/com/hortonworks/streamline/webservice/StreamlineApplication.java

```diff
@@ -252,7 +253,7 @@ private void registerResources(StreamlineConfiguration configuration, Environmen
             authorizerConfig.put(DefaultStreamlineAuthorizer.CONF_CATALOG_SERVICE, securityCatalogService);
             authorizerConfig.put(DefaultStreamlineAuthorizer.CONF_ADMIN_PRINCIPALS, authorizerConf.getAdminPrincipals());
             authorizer.init(authorizerConfig);
-            environment.jersey().register(new StreamlineBasicAuthorizationRequestFilter());
+            environment.jersey().register(new StreamlineKerberosRequestFilter());
```

Comment:
Does this add the filter at the end in filter chain? If so we are okay. If not we need to make sure the AuthenticationFilter is executed before this one